Dockerfile

```diff
@@ -21,6 +21,10 @@ RUN mkdir -p /srv/www /root /var/run/s6 /etc/services.d/chromeheadless && \
     apt-get -qy install \
         --no-install-recommends \
         google-chrome-unstable && \
+    # Install roboto fonts from deb repository
+    apt-get -qy install fonts-roboto && \
+    # Replace selected roboto fonts with newer versions from google
+    ls -la /usr/share/fonts/truetype/roboto | grep .ttf | grep -v "Condensed" | awk '{print "wget -q https://github.com/google/fonts/blob/master/apache/roboto/"$9" -O /usr/share/fonts/truetype/roboto/"$9}' | sh && \
```

Comment:
I think verbosity is the better part of valour here; it doesn't matter for the final image size, so I think a straightforward loop is better.

```
cd /usr/share/fonts/truetype/roboto && \
  for variant in $(ls *.ttf | grep -v Condensed); do \
    rm -f "./${variant}"; \
    wget -q "https://github.com/google/fonts/blob/master/apache/roboto/${variant}"; \
  done